console/src/components/Services/Data/TableModify/ColumnEditorList.js

```diff
@@ -104,6 +105,31 @@ const ColumnEditorList = ({
 
       const isOk = getConfirmation(confirmMessage, true, colName);
       if (isOk) {
+        const pk = [];
+        let flag = true;
+        for (const [_, value1] of tableSchema.primary_key.columns.entries()) {
+          for (const [index2, value2] of tableSchema.columns.entries()) {
```

Comment:
Could you try and do the same with `.forEach` since we're using to array's here?

Reply:
Yeah. I will try doing with .forEach